components/prize_pool/commons/prize_pool_import.lua

```diff
@@ -137,12 +141,24 @@ end
 
 -- Compute placements and their entries from a GroupTableLeague record.
 function Import._computeGroupTablePlacementEntries(standingRecords, options)
+	local isSwiss = Import._isSwissStage(standingRecords)
 	local needsLastVs = Import._needsLastVs(standingRecords)
 	local placementEntries = {}
 	local placementIndexes = {}
 
+	local lastEntry = nil
+	local lastPlacement = nil
 	for _, record in ipairs(standingRecords) do
 		if options.isFinalStage or Table.includes(options.groupElimStatuses, record.currentstatus) then
+			-- Only discriminate placement in Swiss by series score.
```

Comment:
not a fan of it
there actually might be cases where swiss should look at the map/game scores etc
imo it would be way better to let swissable league sort this out
as in letting it store something in extradata that tells which placement it should be for the import
but thats just my 2 cents, maybe @Rathoz has a different view on things

Reply:
If you know a different way to do it, I'd be glad to hear it, like I mentioned in the PR

Reply:
as said above and on discord imo it should be done via swiss table league
but it probably would be good to do it with the (match2 based) rewrite martin wants to do